.canon.yaml

```diff
@@ -0,0 +1,8 @@
+viam-cartographer:
+    image_amd64: ghcr.io/viamrobotics/canon:amd64
+    image_arm64: ghcr.io/viamrobotics/canon:arm64
+    minimum_date: 2022-12-20T18:49:25.781750658Z
+    update_interval: 168h0m0s
```

Comment:
is this fine to leave as 1 week? what will happen to inactive persistent instances if a week has past?

Reply:
So this is only how often it will check for updates to the IMAGES... if/once an update is downloaded, it'll then start warning any time you enter a persistent container that is out of date (but leaves it up to you to terminate it and start a new one when you're ready.)